### ISSUE TITLE
Esophageal epithelial atrophy and hyperplasia

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10042]
**Preferred term label:**
Esophageal epithelial atrophy
AND
Esophageal epithelial hyperplasia

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Both are present in patches in Plummer-Vinson syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
similar to https://github.com/obophenotype/human-phenotype-ontology/issues/10041
it does not make sense to have a separate term for this since it is a component of Esophageal web that is found in only one disease (AFAIK). Please reopen if I am mistaken.
